js/components/frame.js

```diff
@@ -881,10 +881,31 @@ class Frame extends ImmutableComponent {
           method = (currentDetail, originalDetail) =>
             windowActions.setAutofillCreditCardDetail(currentDetail, originalDetail)
           break
+        case messages.TORRENT_MESSAGE:
+          // Relay torrent IPC from the webview to a browser process
+          method = (message) => {
+            if (typeof message.channelID !== 'string') {
+              throw new Error('Invalid or missing channelID: ' + JSON.stringify(message))
+            }
+            if (this.torrentChannelID && this.torrentChannelID !== message.channelID) {
+              throw new Error('ChannelID changed, expected ' + this.torrentChannelID +
```

Comment:
What happens if I reload the tab? Then the webview will select a new `channelID` and this will throw an error. We should probably just always set `this.torrentChannelID = message.channelID`, right?